client/java/src/main/java/io/openlineage/client/circuitBreaker/NoOpCircuitBreaker.java

```diff
@@ -0,0 +1,38 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.circuitBreaker;
+
+import java.util.concurrent.Callable;
+import lombok.extern.slf4j.Slf4j;
+
+/** Non-closing circuit breaker which always runs callable. */
+@Slf4j
+public class NoOpCircuitBreaker extends CommonCircuitBreaker {
+
+  public NoOpCircuitBreaker() {
+    super(0);
+  }
+
+  @Override
+  public boolean isClosed() {
+    return false;
+  }
+
+  @Override
+  public <T> T run(Callable<T> callable) {
+    try {
+      return callable.call();
+    } catch (Exception e) {
+      log.error("OpenLineage callable failed to execute. Swallowing the exception {}", e);
```

Comment:
Swallowing the exception is not exactly no-op, do we really want that?

Reply:
This can happen in case of some uncaught exception of OL integration. This should not happen, but if this is happens we should catch it. I think we want this.